src/main/java/runQuiz/RunQuizPresenter.java

```diff
@@ -60,8 +60,6 @@ public void prepareSuccessView(ShowProblemOutputData outputData) {
             viewBound.updateView(viewModel);
         }
 
-        System.out.println("Not sure how we got here");
```

Comment:
Thanks for removing this. Albert just did not know what was happening

Reply:
Oh yeah, sorry about that, that was a remnant of back when there were return statements above that line.